src/components/Room/Room.tsx

```diff
@@ -32,14 +35,27 @@ export default function Room() {
   const classes = useStyles();
   const { isChatWindowOpen } = useChatContext();
   const { isBackgroundSelectionOpen } = useVideoContext();
+  const { isGridModeActive } = useAppState();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+
   return (
     <div
       className={clsx(classes.container, {
         [classes.rightDrawerOpen]: isChatWindowOpen || isBackgroundSelectionOpen,
       })}
     >
-      <MainParticipant />
-      <ParticipantList />
+      <ParticipantAudioTracks />
```

Comment:
I'm wondering if we should have a comment explaining `<ParticipantAudioTracks />` and why it is necessary for grid mode? I see you left one in `Publication.tsx` but I'm wondering if others will be confused as to why audio tracks aren't treated the same was as video tracks. Just a thought 😅

Reply:
Good idea! I added a comment.